original/java/src/test/java/PageDBTest.java

```diff
@@ -0,0 +1,12 @@
+import org.dbpedia.categories.database.PageDB;
+
+public class PageDBTest {
+
+	public static void main(String[] argv){
```

Comment:
Maybe I should concentrate on this part after I make the workflow working. I just found that Eclipse IDE has few methods to generate these. 
I will remove this file and make a commit.